certbot-dns-sakuracloud/docs/api/dns_sakuracloud.rst

```diff
@@ -0,0 +1,5 @@
+:mod:`certbot_dns_sakuracloud.dns_sakuracloud`
+----------------------------------
```

Comment:
ReStructuredText is a bit picky. You'll need to add more hypen characters so that the length of this line matches the one above. This will eliminate the following warning while building the documentation:
```
/certbot/certbot-dns-sakuracloud/docs/api/dns_sakuracloud.rst:2: WARNING: Title underline too short.

:mod:`certbot_dns_sakuracloud.dns_sakuracloud`
```